src/cmd/ksh93/include/shell.h

```diff
@@ -414,8 +408,6 @@ extern Shell_t sh;
 #endif
 #endif
 #ifndef _SH_PRIVATE
-#undef access
-#define access(a, b) sh_access(a, b)
```

Comment:
I suspect it might break https://github.com/att/ast/blob/8d69ea8fdc6c02d317afac76bd6b26f3915e86e9/src/lib/libcmd/mkdir.c#L148 if effective and real user id is different.

Reply:
Also we should probably replace https://github.com/att/ast/blob/8d69ea8fdc6c02d317afac76bd6b26f3915e86e9/src/cmd/ksh93/sh/path.c#L1016 by manually calling `sh_access`.

Reply:
Note that neither of those use that macro. I verified this is safe by changing it to `#define access(a, b) WTFsh_access(a, b)`. If there were any places using that macro that change would have broken the build by introducing an unresolved symbol. Whether those two uses should be using `sh_access` and they are currently broken is not clear. The answer for the *path.c* instance is almost certainly yes since there are two places in that module which use `sh_access`.

Reply:
The `access()` call in *mkdir.c* is suspicious in as much as it looks to be pointless. In any case I don't think it needs to be changed to `sh_access()`. The one in *path.c*, on the other hand, does look like it should be using `sh_access()` so I'll change that one.